rep-0003.rst

```diff
@@ -250,6 +310,13 @@ consulting Ubuntu's release and end-of-life schedule [1]_.
 
 These targets, including starting and ending support dates, are based on the Distribution Timeline to meet minimum requirements. [3]_
 
+Architectures
+-------------
+
+As of ROS Lunar, we do not build packages for x86 architectures.
```

Comment:
At least [Wikipedia/x86](https://en.wikipedia.org/wiki/X86) (I know, I know) still uses the moniker `x86` for the 64 bit versions of those processors (ie: `amd64`). May be my age, but I had to read this sentence twice to make sure I understood it correctly (ie: x86 based archs are still supported, but only the `amd64` variants of them).